python/dgl/distributed/graph_partition_book.py

```diff
@@ -149,63 +167,71 @@ def metadata(self):
             Meta data of each partition.
         """
 
-    def nid2partid(self, nids):
+    def nid2partid(self, nids, ntype):
         """From global node IDs to partition IDs
 
         Parameters
         ----------
         nids : tensor
             global node IDs
+        ntype : str
+            The node type
 
         Returns
         -------
         tensor
             partition IDs
         """
 
-    def eid2partid(self, eids):
+    def eid2partid(self, eids, etype):
         """From global edge IDs to partition IDs
 
         Parameters
         ----------
         eids : tensor
             global edge IDs
+        etype : str
+            The edge type
```

Comment:
Just to make sure: what to do with canonical edge types (i.e. triplets)?  It's OK not handling them though.

Reply:
actually, one TODO I have is that the current API doesn't support canonical edge types. Will it be quick to support it?

Reply:
Not sure.  You can leave it for now.